vllm_hpu_extension/ops.py

```diff
@@ -241,12 +241,8 @@ def __init__(self):
     def set_weight(self, w):
         self.weight = w
```

Comment:
Is there any use to `self.weight` now?

Reply:
yes, during quantization we quantize this bf16 weight to fp8